draft-terbu-sd-jwt-vc.md

```diff
@@ -127,8 +124,10 @@ document are to be interpreted as described in RFC 2119 [@!RFC2119].
 ## Terms and Definitions
 
 This specification uses the terms "Holder", "Issuer", "Verifier", defined by
-[@!I-D.ietf-oauth-selective-disclosure-jwt], Verifiable Credential defined by
-[@VC-DATA].
+[@!I-D.ietf-oauth-selective-disclosure-jwt].
+
+Verifiable Credential (VC):
+:  An Issuer-signed Credential whose authenticity can be cryptographically verified.
```

Comment:
```suggestion
:  An Issuer-signed statement about a Subject whose authenticity can be cryptographically verified.
```

Reply:
I'm ok with this change but we should not introduce Credential since we don't have a definition for that. I'll replace with the definition above.